bindings/wasm/identity_wasm/examples/src/main.ts

```diff
@@ -61,6 +62,8 @@ export async function main(example?: string) {
             return await advancedTransaction();
         case "12_iota_keytool_integration":
             return await iotaKeytoolIntegration();
+        case "0_did_owns_did":
+            return await didOwnsDid();
```

Comment:
Shouldn't this be between `7_revoke_vc` and `4_custom_resolution`, as there's the jump from basic to advanced? ^^